src/imagery/i.saocom/i.saocom.import/i.saocom.import.html

```diff
@@ -0,0 +1,65 @@
+SLC products are complex matrices, but GRASS will import them as two separate bands containing the real and imaginary parts. This structure allows for further manipulation and prevents from losing important information. 
+
+<p>
+By default <em>i.saocom.import</em> will process all the available polarizations, which can be selected with the <b>pols</b> option. 
+The module assumes by default that the data is zipped but this can be changed with the <b>is_zip</b> option. 
+
+<p>
+The user can also apply a multilooking factor, which must be indicated as a list, followinf the order <em>[az_loos,rg_looks]</em>.
+By default no multilooking is applied.
+
+<p>
+The module requires to specify a <b>basename</b> value which will be used as prefix for all the polarization bands to be imported. Additionaly, a Ground Control Points file
+will associated to this basename will be created within de GRASSDATA directory, in case the user performs later geocoding with <em>i.saocom.geocode</em>. This GCP information is extracted from the original SAOCOM-1 matrix using Python <em>rasterio</em> library and will account for any multilooking applied to the images.
+
+<h2>NOTES</h2>
+
+SAOCOM L1A images are projected in radar coordinates. To run this module it is important to previously create an XY (unprojected) location.
+A file named <b>GCPS.csv</b> with the Ground Control Points information will be stored in the current location's <b>cell_misc</b> directory, inside a sub-folder that will be named after the <b>basename</b> value.
+
+<h2>EXAMPLES</h2>
+
+<h3>Import all polarizations</h3>
+
+Firstly, an unprojected (XY) location must be created, the example below creates a location called <em>saocom</em>:
+
+<div class="code"><pre>
+grass -c XY saocom -e
+</pre></div>
+
+Then, run <em>i.saocom.import</em> from a zip file containing pols = ['hh','hv','vh','vv']:
+
+<div class="code"><pre>
+zip_file = 'S1B_OPER_SAR_EOSSP__CORE_L1A_OLF_20211225T165228.zip'
+i.saocom.import data=zip_file is_zip='yes' basename='SAO1B_20211222'
+</pre></div>
+
+The previous example will create two bands using the given <b>basename</b> as prefix: <em>SAO1B_20211222_[pol]_r</em> (real band) <em>SAO1B_20211222_[pol]_i</em> (imaginary band). <em>[pol]</em>
+indicates the imported polarization, which for SAOCOM images can be any of ['hh','hv','vh','vv'].
+
+<h3>Import specific polarizations and apply multilooking</h3>
+
+<div class="code"><pre>
+zip_file = 'S1B_OPER_SAR_EOSSP__CORE_L1A_OLF_20211225T165228.zip'
+i.saocom.import data=zip_file is_zip ='yes' basename='SAO1B_20211222' pols=['hh','vv'] multilook=[4,2]
+</pre></div>
+
+The previous example will import only the <em>hh</em> and <em>vv</em> vv polarizations, and in case none of them are found the program will let the user know which are the available polarization channels in the dataset. The multilooking factor will be of 4 in azimuth direction and 2 in range direction.
```

Comment:
```suggestion
The previous example will import only the <em>hh</em> and <em>vv</em> polarizations, and in case none of them are found the program will let the user know which are the available polarization channels in the dataset. The multilooking factor will be of 4 in azimuth direction and 2 in range direction.
```